eng/build.sh

```diff
@@ -377,6 +378,18 @@ while [[ $# > 0 ]]; do
       shift 2
       ;;
 
+     -keepsymbols)
+      if [ -z ${2+x} ]; then
+        echo "No value for keep symbols is supplied. See help (--help) for supported values." 1>&2
+        exit 1
+      fi
+      passedKeepSymbols="$(echo "$2" | awk '{print tolower($0)}')"
+      if [ "$passedKeepSymbols" = true ]; then
+        cmakeargs="${cmakeargs} -cmakeargs -DCLR_KEEP_SYMBOLS=true"
```

Comment:
I would prefer creating a msbuild property for this instead of passing it in the cmakeargs. We do that e.g. for the `-portablebuild` option, so I'd like to use unified way for all of them. You would then add this option to `eng/build-commons.sh`, which is used to parse the common options for coreclr, libraries and installer .sh scripts and add the cmake arg there. You'll need to update src/coreclr/runtime.proj, src/installer/corehost/corehost.proj and src/libraries/Native/build-native.proj to convert the msbuild property to the -keepsymbols option passed to the respective .sh scripts.

Reply:
Does the updated patch match what you had in mind?